src/Playwright/Transport/WebSocketTransport.cs

```diff
@@ -104,13 +104,17 @@ private async Task DispatchIncomingMessagesAsync()
                     if (_webSocket.State != WebSocketState.Open)
                         break;
 
-                    memoryStream.Write(buffer.Array, 0, result.Count);
+#pragma warning disable CA1835 // Change the 'WriteAsync' method call to use the 'Stream.WriteAsync(ReadOnlyMemory<byte>, CancellationToken)' overload
```

Comment:
Same, if it's due to many versions, let's add it to the ruleset.

Reply:
(we eventually get rid of this file soon, so I'd keep it for now)